### DISCUSSION TITLE
Eventual Playback support

### DISCUSSION THREAD

**Discussion by jonbng** [browniebroke/deezer-python#694] [Ideas]
Because of another project I need to write TOS and Guideline complaint playback support in python. I was thinking I might make it something separate so other people also could use it. So my idea was to add it to this library. Do you want me to do that or would your rather want me to make it a seperate thing?

**Comment by browniebroke** [comment]
I am sorry, I don't understand what you're suggesting. Could you expand please?

**Reply by jonbng** [reply]
Right now this library doesn't support streaming songs. 
I am going to write a library for streaming music from deezer. I am suggesting making it a part of deezer-python. 
I am asking about your thoughts on that. Would you rather not have me add it to deezer-python because it's not using the official api

**Reply by browniebroke** [reply]
I didn't know that was possible, I thouight they prevented this from being possible, there might be some legal concerns at play.  Also, generally speaking the more something uses unoffiicial APIs, the more it's likely to break.

Given all that, I'd be against including it here, but that being said, I'd be interested to see how it works out for you.

**Reply by jonbng** [reply]
OK that's completely understandable. Also kinda what i expected :smile: Thanks anyway
